data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/special_calculations.cfg

```diff
@@ -213,9 +213,9 @@
 # API(s) being tested: [attacks]multiply=
 ##
 # Expected end state:
-# Bob's attack has 9 strikes, because 3 * 3.334 first rounds 3.334 to 3.33, and then 3 * 3.33 floors to 9.
+# Bob's attack has 10 strikes, because 3 * 3.334 first rounds 3.334 to 3.33, and then 3 * 3.33 floors to 10.
```

Comment:
This comment is out of date. I'm getting back to reviewing, so haven't checked whether the truncation of 3.334 to 3.33 still exists, but here "floors" means "rounds down".

Reply:
changed to "rounds up"

Reply:
But does it still round in the middle, or not?

This one might still do, IIRC it was decimal truncation so probably still working on string values at that point.